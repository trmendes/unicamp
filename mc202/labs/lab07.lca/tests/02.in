20 33
link 2 1
link 3 2
link 4 3
link 5 4
link 6 5
link 7 6
link 8 7
link 9 8
link 10 9
link 11 10
link 12 11
link 13 12
link 14 13
link 15 14
link 16 15
link 17 16
link 18 17
link 19 18
link 20 19
lca 20 19
lca 19 20
lca 10 20
lca 1 19
lca 1 1
lca 1 2
lca 2 3
lca 5 6
lca 4 18
lca 3 7
lca 3 4
lca 7 7
lca 2 1
lca 1 6
