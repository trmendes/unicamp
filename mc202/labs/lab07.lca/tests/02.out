19
19
10
1
1
1
2
5
4
3
3
7
1
1
